packages/fetchai/protocols/fipa/dialogues.py

```diff
@@ -60,53 +59,48 @@
     ],
 }  # type: Dict[FipaMessage.Performative, List[Union[None, FipaMessage.Performative]]]
 
+SUPPLY_DATAMODEL_NAME = "supply"
+DEMAND_DATAMODEL_NAME = "demand"
```

Comment:
This needs to be defined elsewhere (`custom_types.py` for instance). It can not be generated...

Reply:
Done